tests/eventdispatcher/PatientFilterEventHookTest/Module.php

```diff
@@ -109,7 +108,7 @@ public function filterPatientFinderByBlacklist(PatientFinderFilterEvent $event)
 
         // If there are patients to hide from this user, build a filter
         if (count($patientsToHide)) {
-            $patientsToHideEscaped = array_map( function($elem) {
+            $patientsToHideEscaped = array_map(function($elem) {
```

Comment:
@bradymiller @adunsulag CI build doesn't like this line, but looks inline with your standards. I think it doesn't like the space before the function's first curly brace?

Reply:
Wouldn't worry to much about this since I can run a quick script (https://gist.github.com/bradymiller/ae5069a058683b5a02c58b6afbcdace8) to automatically fix things after it goes into the codebase. But guessing following will pass:
```
$patientsToHideEscaped = array_map(
    function($elem)
        {
            return add_escape_custom($elem);
        },
        $patientsToHide
    );
```
(just guessing though)